### PR TITLE
Stitch in search results page skeleton

### PR DESCRIPTION
**Pull request by dblandin**
Stitch in search results skeleton within search results page.

Depends on https://github.com/artsy/reaction/pull/2305#

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The staging version of Metaphysics has breaking changes, this might not affect your build, but could mean your queries don't work the way you expect. See below for breaking changes:</td>
    </tr>
  </tbody>
</table>


- :warning: Type 'AnalyticsPartnerStatsType' was removed
- :warning: Type 'AnalyticsPartnerTimeSeriesStatsType' was removed
- :warning: Field 'Query.analyticsPartnerStats' changed type from 'AnalyticsPartnerStatsType' to 'AnalyticsPartnerStats'
- :warning: 'AnalyticsArtworksPublishedTimeSeriesStats' object type no longer implements 'AnalyticsPartnerTimeSeriesStatsType' interface
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 5cbb1f933cff9cbcfe240beea653e6c62c6a573b
</p>

File: src/desktop/components/main_layout/templates/search_results.jade [hunk 1]
```diff
@@ -0,0 +1,8 @@
+extends react_redesign
```
Comment (mzikherman) [inline src/desktop/components/main_layout/templates/search_results.jade]: #minor nitpick incoming:

I'm not sure that this needs an entire layout here, you can probably update `react_redesign` to have a:

```
if loading
  #loading-container
    != loadingComponent
```

in there after the `#react-root`, and continue to pass a `loadingBlock`.

_or_

I would go with a template like https://github.com/artsy/force/blob/72687e28e1d78755376537abad94b6479ea8ccc2/src/desktop/apps/search2/templates/index.jade defined in the app, so it doesn't need to be in this more general desktop components folder and is better co-located with the relevant code.

This commit https://github.com/artsy/force/pull/3895/commits/72687e28e1d78755376537abad94b6479ea8ccc2 is an example of that.

Either of those (my pref is towards the latter), feels better than adding a whole new layout in this folder.

File: src/desktop/apps/search2/server.tsx [hunk 2]
```diff
@@ -1,6 +1,9 @@
 import { stitch } from "@artsy/stitch"
 import express, { Request, Response, NextFunction } from "express"
 import { stringify } from "querystring"
+import { SearchResultsSkeleton } from "reaction/Apps/Search/Components/SearchResultsSkeleton"
+import React from "react"
+import { StitchWrapper } from "desktop/components/react/stitch_components/StitchWrapper"
```
Comment (mzikherman) [inline src/desktop/apps/search2/server.tsx]: ooo good to know about!!

**Comment by mzikherman** [comment]
LGTM! Made one minor comment about the layout thing but not too strongly opinionated- feel free to iterate + merge as needed.

**Pull request merged**